internal/client/client_batch.go

```diff
@@ -107,33 +107,56 @@ func (b *batchCommandsBuilder) push(entry *batchCommandsEntry) {
 	b.entries.Push(entry)
 }
 
-// build builds BatchCommandsRequests and calls collect() for each valid entry.
+const highTaskPriority = 10
+
+func (b *batchCommandsBuilder) hasHighPriorityTask() bool {
+	return b.entries.highestPriority() >= highTaskPriority
+}
+
+// buildWithLimit builds BatchCommandsRequests and calls collect() for each valid entry.
```

Comment:
Better to explain the meaning and usages of `limit` in the comment updates.

Reply:
fixed